tests/Database/Base.php

```diff
@@ -1930,4 +1931,244 @@ public function testGetId()
         // ensure two sequential calls to getId do not give the same result
         $this->assertNotEquals($this->getDatabase()->getId(10), $this->getDatabase()->getId(10));
     }
+
+    public function testUpdateAttribute()
+    {
+        $database = static::getDatabase();
+
+        Structure::addFormat('enum', function($options) {
+            return $options;
+        }, 'string');
+
+        $collection = $database->createCollection('stocks');
+        $database->createAttribute('stocks', 'brand', Database::VAR_STRING, 128, true);
+        $database->createAttribute('stocks', 'price', Database::VAR_STRING, 128, false);
+        $database->createAttribute('stocks', 'authors', Database::VAR_STRING, 128, true);
+        $database->createAttribute('stocks', 'employee', Database::VAR_STRING, 128, true);
+        $database->createAttribute('stocks', 'unsignedAttribute', Database::VAR_INTEGER, 0, true);
+
+        $database->createIndex('stocks', 'index1', Database::INDEX_KEY, ['brand'], [128], [Database::ORDER_ASC]);
+
+        $database->createDocument('stocks', new Document([
+            '$read' => ['role:all'],
+            '$write' => ['role:all'],
+            'brand' => 'Apple',
+            'price' => '171',
+            'authors' => 'Tim Cook',
+            'employee' => '{}',
+            'unsignedAttribute' => -5
+        ]));
+
+        $database->createDocument('stocks', new Document([
+            '$read' => ['role:all'],
+            '$write' => ['role:all'],
+            'brand' => 'Microsoft',
+            'price' => '299',
+            'authors' => 'Satya Nadella',
+            'employee' => '{}',
+            'unsignedAttribute' => -50
+        ]));
+
+        $collection = $database->getCollection('stocks');
+        $attributes = $collection->getAttribute('attributes');
+
+        $this->assertEquals(128, $attributes[0]->getAttribute('size'));
+        $this->assertEquals(true, $attributes[0]->getAttribute('required'));
+        $this->assertEquals(null, $attributes[0]->getAttribute('default'));
+        $this->assertEquals(Database::VAR_STRING, $attributes[1]->getAttribute('type'));
+        $this->assertEquals(true, $attributes[1]->getAttribute('signed'));
+        $this->assertEquals(false, $attributes[2]->getAttribute('array'));
+        $this->assertEquals(null, $attributes[3]->getAttribute('format'));
+        $this->assertIsArray($attributes[3]->getAttribute('formatOptions'));
+        $this->assertCount(0, $attributes[3]->getAttribute('formatOptions'));
+
+        $this->assertTrue($database->updateAttribute('stocks', 'brand',
+            null, // type
+            10, // size
+            false, // required
+            "Unnamed", // default
+            null, // signed
+            null, // array
+            null, // format
+            null, // formatOptions
+            null, // filters
+        ));
```

Comment:
For readability we can use named arguments here:
```suggestion
        $this->assertTrue($database->updateAttribute('stocks', 'brand',
            size: 10,
            required: false,
            default: "Unnamed"
        ));
```